db/query_test.go

```diff
@@ -641,7 +641,7 @@ func TestQueryChannelsActiveOnlyWithLimit(t *testing.T) {
 		require.NoError(t, err, "Couldn't create document")
 		require.Equal(t, "3-a", revId, "Couldn't create tombstone revision")
 
-		docIdFlagMap[doc.ID] = uint8(0x14) // 20 = Branched (16) + Hidden(4)
+		docIdFlagMap[doc.ID] = channels.Branched + channels.Hidden // 20 = Branched (16) + Hidden(4)
```

Comment:
The standard approach in the SG codebase for combining bit flags is to use bitwise OR.  Functionally equivalent for distinct bit values, but I think a better approach that avoids accidental addition of non-distinct flags.  Same suggestion for the usages below.   
```suggestion
		docIdFlagMap[doc.ID] = channels.Branched|channels.Hidden // 20 = Branched (16) + Hidden(4)
```

Reply:
Added changes to combine bit flags using bitwise OR to avoid accidental addition of non-distinct flags.